src/TLuaInterpreter.cpp

```diff
@@ -5800,6 +5800,20 @@ int TLuaInterpreter::getMainWindowSize( lua_State *L )
     return 2;
 }
 
+int TLuaInterpreter::getMousePosition( lua_State *L )
+{
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    //QSize mainWindowSize = pHost->mpConsole->getMainWindowSize();
+    //QPoint pos = widget->mapFromGlobal(QCursor::pos());
+    QPoint pos = pHost->mpConsole->mapFromGlobal(QCursor::pos());
```

Comment:
Are we absolutely sure that `pHost` and `pHost->mpConsole` are definitely valid (non-NULL) under any situation where this code might be used (loading of lua sub-systems as part of "auto-loading" of a profile for instance).   For the former you might want the sort of boiler-plate check that I do elsewhere in this class - search for the "NULL" word in this file...! :wink: